### ISSUE TITLE
raven.captureMessage should not throw in dev environment

### ISSUE THREAD

**Issue by mydea** [damiencaselli/ember-cli-sentry#131]
In our app, we use `captureMessage()` in some places to basically "log" data to sentry.
However, it was unexpected for us that in development (e.g. when sentry is not usable), it re-throws the given message, leading to code aborting.

This happens here: https://github.com/damiencaselli/ember-cli-sentry/blob/aaf3f97a6e43b54dc3304a749f0b4ee0d7ac3440/addon/services/raven.js#L148

```js
captureMessage(message) {
    if (this.get('isRavenUsable')) {
      Raven.captureMessage(...arguments);
    } else {
      throw new Error(message);
    }
    return true;
  }
```

This basically means that the functionality is widely different when raven is usable or not, as it will not throw in production, but throw in development. It leads to us wrapping all calls to `raven.captureMessage` in a try-catch block, which I guess is not the idea here.

Wouldn't it make more sense to just `console.warn(message)` as a fallback? It even says so in the comment above the function, so maybe it is just a copy paste error?

**Comment by mydea** [comment]
Ah, I just saw there is already a PR for that: https://github.com/damiencaselli/ember-cli-sentry/pull/97.

**Comment by Turbo87** [comment]
yeah, I think that makes sense. I'll put review of #97 on my todo list.

**Comment by Turbo87** [comment]
resolved by #97
